UI/CUIControls.cpp

```diff
@@ -1924,3 +1928,119 @@ void RotatingGraphic::Render() {
     glPopClientAttrib();
     glPopMatrix();
 }
+
+////////////////////////////////////////////////
+// CUIToggleButton
+////////////////////////////////////////////////
+CUIToggleRepresenter::CUIToggleRepresenter(boost::shared_ptr<GG::SubTexture> icon,
+                                           const GG::Clr& highlight_clr) :
+                                           m_unchecked_icon(icon),
```

Comment:
indent initializer list one tab from class name, not to be in line with parameters